benefits/core/session.py

```diff
@@ -148,17 +148,17 @@ def logged_in(request):
 
 def logout(request):
     """Reset the session claims and tokens."""
-    update(request, oauth_claim=False, oauth_token=False, enrollment_token=False)
+    update(request, oauth_claims=False, oauth_token=False, enrollment_token=False)
```

Comment:
I know you didn't introduce this, but I'm wondering if `False` makes sense here, especially now with the list approach? Should we set this to an empty list `[]` instead?

Reply:
Yep, makes sense to change it to `[]`, changed in c0a2516db02df7dea3467262321d145e5117d8b4